src/main/resources/openapi/EMSIntegrationAPIs.yaml

```diff
@@ -821,12 +821,12 @@ components:
       properties:
         notes:
           type: string
-          maxLength: 1024
+          maxLength: 4000
```

Comment:
do we need to increment the version of this spec?

Reply:
Done